test/torchaudio_unittest/functional/functional_cpu_test.py

```diff
@@ -70,47 +70,104 @@ def test_pitch(self, frequency):
         self.assertFalse(s)
 
 
-class TestDB_to_amplitude(common_utils.TorchaudioTestCase):
-    def test_DB_to_amplitude(self):
-        # Make some noise
-        x = torch.rand(1000)
+class Testamplitude_to_DB(common_utils.TorchaudioTestCase):
+    AMPLITUDE_MULT = 20.
+    POWER_MULT = 10.
+    AMIN = 1e-10
+    REF = 1.0
+    # FIXME: So the DB multiplier is 0?
+    DB_MULT = math.log10(max(AMIN, REF))
+
+    def _make_spectrogram(self, items, channels, wave_samples, scale=1.):
```

Comment:
This method does not seem to reference the class or an instance. Please make it a plain function. (if you end up keeping this)

Reply:
Also looking at the individual test case, the resulting spectrograms are modified. That makes me wonder if it is worth introducing such helper function.

Reply:
I've removed this method. I'm just generating random tensors now - it does seem cleaner.